.github/workflows/cicd.yml

```diff
@@ -0,0 +1,49 @@
+name: NodeJS with Webpack
+
+on:
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read # for checkout
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [20.11.x]
```

Comment:
How about 20.x ?

Reply:
<img width="1038" alt="스크린샷 2024-02-01 오후 3 51 50" src="https://github.com/JunbaeJs/junbae-mode/assets/152782031/5d74ef56-ef69-4227-9cb2-dbe4ff538b01">
we require Node version 20.8.1 or higher. and LTS is 20.11.0